src/main/java/teammates/ui/controller/StudentProfilePictureEditAction.java

```diff
@@ -76,104 +74,100 @@ protected ActionResult execute() throws EntityDoesNotExistException {
      * @throws IOException
      * TODO: use the function 'writeDataToGcs' in GoogleCloudStorageHelper to achieve this 
      */
-    private void uploadFileToGcs(byte[] transformedImage)
-            throws IOException {
-        
+    private void uploadFileToGcs(byte[] transformedImage) throws IOException {
         GcsFilename fileName = new GcsFilename(Config.GCS_BUCKETNAME, account.googleId);
-        
-        GcsService gcsService = GcsServiceFactory.createGcsService(RetryParams.getDefaultInstance());
-        GcsOutputChannel outputChannel =
-                gcsService.createOrReplace(fileName, new GcsFileOptions.Builder().mimeType("image/png").build());
-        
+
+        GcsService gcsService = GcsServiceFactory.createGcsService(RetryParams
+                .getDefaultInstance());
```

Comment:
Indent this line so that the `.` in this line appears to the right of the `.` in the line above? reason: This dot is part of the parameter of the dot above.
Or just get rid of the line break.